src/main/java/com/google/sps/data/WaterSystem.java

```diff
@@ -59,14 +90,14 @@ public WaterSystem(String pwsid) {
                 this.county = cells[46];
```

Comment:
This has potential to cause a lot of bugs in future. Also it is not clean. Move these number to static const?